libs/upd/views/tasks/src/lib/tasks-home/tasks-home.component.ts

```diff
@@ -30,62 +30,138 @@ export class TasksHomeComponent implements OnInit {
   ngOnInit() {
     this.tasksHomeService.init();
 
+    //     _id(pin):"621d280492982ac8c344d19b"
+    // airtable_id(pin):"rec0lmQEXjRMbrLQJ"
+    // title(pin):"Access my Tax Information"
+    // group(pin):"Check Status"
+    // subgroup(pin):""
+    // topic(pin):"Taxes"
+    // subtopic(pin):""
+    // program(pin):""
+    // service(pin):"E-services (online portals, etc.)"
+    // user_journey(pin):
+    // status(pin):""
+
     combineLatest([this.tasksHomeData$, this.currentLang$]).subscribe(
```

Comment:
Not necessary per se, but would be good to change this to an observable with `.pipe()` + `async` pipe in the template.

This is for a few reasons:

1. `.subscribe()` should be avoided wherever possible for reasons I'm sure you're aware of.
2. `combineLatest()` emits a value whenever any of the observables emit, which means it'll emit twice on an initial load. With subscribe, you're reassigning `this.columns` any time a value emits, which means the component will re-render. With the `async` pipe, there's some caching going on under the hood so the re-render can be avoided. (Though a piped observable would probably create a new array too, so depending on how the caching works it may not actually make a difference, in which case selectors would need to be used).

That said, in this specific scenario, the Tasks home table only has a few hundred entries so most performance optimizations probably have little impact, **but** it's just generally good practice to try and convert any `.subscribe()`s that you come across where it's an easy switch and you're making changes anyways.